WarpBuf/src/WarpBufChugin.cpp

```diff
@@ -544,18 +248,20 @@ void WarpBufChugin::tick(SAMPLE* in, SAMPLE* out, int nframes)
 
     //// copy from buffer to output.
     // out needs to receive alternating left/right channels.
-    for (int chan = 0; chan < channels; chan++) {
+    for (int chan = 0; chan < m_channels; chan++) {
         auto chanPtr = m_retrieveBuffer[chan];
         for (int i = 0; i < nframes; i++) {
             out[chan + 2 * i] = *chanPtr++;
         }
     }
 }
 
-bool WarpBufChugin::read(const string& path) {
-
-    m_fileWasRead = false;
+// return true if the file was read
+bool
```

Comment:
make one line?

Reply:
It's hard to tell on the GitHub interface which line you're referring to.

Reply:
Line 226-227 is splitting the return type and the rest of the function declaration for WarpBufChugin::read into two lines